src/compose/types/page-block/geometry/page-block.ts

```diff
@@ -0,0 +1,61 @@
+import { PageBlock, Registry } from '../base'
+import Feed, { FeedInput } from './feed'
+import { RecordFeed } from './feed-record'
+
+const kind = 'Geometry'
+
+interface Bounds {
+  _southWest: {
+    lat: number;
+    lng: number;
+  };
+  _northEast: {
+    lat: number;
+    lng: number;
+  };
+}
+
+class GeometryOptions {
+    public defaultView = ''
+    public center: Array<number> = [30, 30]
+    public feeds: Array<Feed> = []
+    public zoomStarting = 3
+    public zoomMin = 1
+    public zoomMax = 18
+    public bounds: Bounds | null = null
+    public lockBounds = false
+}
+
+export class PageBlockGeometry extends PageBlock {
+  readonly kind = kind
+  public options = new GeometryOptions()
+
+  static feedResources = Object.freeze({
+    record: 'compose:record',
+  })
+
+  constructor (i?: PageBlock | Partial<PageBlock>) {
+    super(i)
+    this.applyOptions(i?.options as Partial<GeometryOptions>)
+  }
+
+  applyOptions (o?: Partial<GeometryOptions>): void {
+    if (!o) return
+
+    this.options.feeds = (o.feeds || [])
```

Comment:
No need for this, you can just use the Apply method like other blocks use (Comment for example), its either the defaults or the ones we get. So no need to fallback